component/tests/golden/apiserver/appcat/appcat/10_appcat_ha_monitoring.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: appcat-ha
+  namespace: syn-appcat
+spec:
+  groups:
+    - name: appcat-ha
+      rules:
+        - alert: AppCatHighAvailableDeploymentWarning
+          annotations:
+            description: The deployment {{ $labels.deployment }} in namespace {{ $labels.namespace
+              }} has less replicas than expected.
+            runbook_url: https://kb.vshn.ch/app-catalog/how-tos/appcat/AppCatHAWarning.html
+            summary: AppCat service instance has unavailable pods.
+          expr: kube_deployment_status_replicas{namespace=~"vshn-(postgresql|redis)-.*"}
+            > 1 AND kube_deployment_status_replicas{namespace=~"vshn-(postgresql|redis)-.*"}
+            - kube_deployment_status_replicas_ready{namespace=~"vshn-(postgresql|redis)-.*"}
+            > 0
+          for: 1m
+          labels:
+            severity: warning
+            syn_team: schedar
+        - alert: AppCatHighAvailableStatefulsetWarning
+          annotations:
+            description: The statefulset {{ $labels.statefulset }} in namespace {{
+              $labels.namespace }} has less replicas than expected.
+            runbook_url: https://kb.vshn.ch/app-catalog/how-tos/appcat/AppCatHAWarning.html
+            summary: AppCat service instance has unavailable pods.
+          expr: kube_statefulset_status_replicas{namespace=~"vshn-(postgresql|redis)-.*"}
+            > 1 AND kube_statefulset_status_replicas{namespace=~"vshn-(postgresql|redis)-.*"}
+            - kube_statefulset_status_replicas_ready{namespace=~"vshn-(postgresql|redis)-.*"}
+            > 0
```

Comment:
do we have both statefulset and deployment in both namespaces?

Reply:
No we don't, why do you ask?

Reply:
then we don't need such alert right?

Reply:
I'm OK with keeping it like this, as it would make it more complicated otherwise and doesn't do any harm

Reply:
it doesn't matter, the alerts are per cluster not per service. One alert that will check in all instance namespaces.